### PR TITLE
docs: Add section on building rocprofsys for no GPU

### PR DESCRIPTION
**Pull request by peterjunpark**
Adds brief information in `install.rst` regarding
- Provided build scripts
- What to do when building rocprofiler-systems w/o ROCm
- Recent changes to the configuration settings as a result of rocprofiler-sdk support.

Fixes minor formatting issues in `install.rst` resulting in Sphinx build warnings

### CODE REVIEW

**Comment by peterjunpark** [comment]
preview: https://advanced-micro-devices-demo--57.com.readthedocs.build/projects/rocprofiler-systems/en/57/install/install.html#rocm-systems-profiler-without-rocm

**Comment by peterjunpark** [comment]
@pranswarup could you take a look?

**Comment by dgaliffiAMD** [comment]
Another option would be to point users to the provided build scripts: 

```
./scripts/build-release.sh --help
Options:
    --core       [+nopython] [+python]                    Core (Use '+nopython' to build w/o python, use '+python' to python build with python)
    --mpi        [+nopython] [+python]                    MPI (Use '+nopython' to build w/o python, use '+python' to python build with python)
    --rocm       [+nopython] [+python]                    ROCm (Use '+nopython' to build w/o python, use '+python' to python build with python)
    --rocm-mpi   [+nopython] [+python]                    ROCm + MPI (Use '+nopython' to build w/o python, use '+python' to python build with python)
    --mpi-impl   [openmpi|mpich]                          MPI implementation

    --lto                  [on|off]                       Enable LTO (default: off)
    --strip                [on|off]                       Strip libraries (default: off)
    --perfetto-tools       [on|off]                       Install perfetto tools (default: on)
    --static-libgcc        [on|off]                       Build with static libgcc (default: on)
    --static-libstdcxx     [on|off]                       Build with static libstdc++ (default: on)
    --hidden-visibility    [on|off]                       Build with hidden visibility (default: on)
    --max-threads          N                              Max number of threads supported (default: 2048)
    --parallel             N                              Number of parallel build jobs (default: 12)
    --generators           [STGZ][DEB][RPM][+others]      CPack generators (default: stgz deb rpm)
```

So, to build without ROCm-support and create a STGZ installer, you would use `./script/build-release.sh --core +python --generators STGZ`.

**Comment by peterjunpark** [comment]
Added https://advanced-micro-devices-demo--57.com.readthedocs.build/projects/rocprofiler-systems/en/57/install/install.html#using-the-build-script and updated https://advanced-micro-devices-demo--57.com.readthedocs.build/projects/rocprofiler-systems/en/57/install/install.html#rocm-systems-profiler-without-rocm

Could you help make sure it's accurate and provide feedback? @dgaliffiAMD @pranswarup Thanks.

**Review by pranswarup — DISMISSED** [review]
Documentation leverages existing build processes and it only adds information on build flags that can be used to build rocprof-sys without rocm or gpu installed.

File: docs/install/install.rst [hunk 1]
```diff
@@ -176,9 +175,9 @@ Optional third-party packages
 * `ROCm <https://rocm.docs.amd.com/projects/install-on-linux/en/latest>`_
 
   * HIP
-  * Roctracer for HIP API and kernel tracing
-  * ROCM-SMI for GPU monitoring
-  * Rocprofiler for GPU hardware counters
+  * ROCTracer for HIP API and kernel tracing
+  * ROCm SMI Lib for GPU monitoring
+  * ROCProfiler for GPU hardware counters
```
Comment (dgaliffiAMD) [inline docs/install/install.rst]:
```suggestion
  * ROCm SMI Lib for GPU monitoring
  * ROCprofiler SDK for GPU hardware counters and ROCm tracing
```
Sorry, updating to reflect some recent changes for the ROCprofiler SDK support.

File: docs/install/install.rst [hunk 2]
```diff
@@ -306,6 +338,20 @@ ROCm Systems Profiler on an application built against OpenMPI causes a segmentat
 This happens because the value of the ``MPI_COMM_WORLD`` is truncated
 during the function wrapping before being passed along to the underlying MPI function.
 
+ROCm Systems Profiler without ROCm
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To build ROCm Systems Profiler for use on systems without a GPU or the ROCm runtime, disable HIP
+support using the CMake configuration option ``ROCPROFSYS_USE_HIP=OFF``. See :ref:`cmake-options`
```
Comment (dgaliffiAMD) [inline docs/install/install.rst]:
```suggestion
To build ROCm Systems Profiler for use on systems without a GPU or the ROCm runtime, disable ROCm
support using the CMake configuration option ``ROCPROFSYS_USE_ROCM=OFF``. See :ref:`cmake-options`
```
Sorry, updating to reflect some recent changes to the configuration settings with the rocprofiler-sdk support.

File: docs/install/install.rst (hunk 1 above)
Comment (peterjunpark) [reply docs/install/install.rst]: Thanks @dgaliffiAMD

File: docs/install/install.rst (hunk 2 above)
Comment (peterjunpark) [reply docs/install/install.rst]: Thanks

Comment (peterjunpark) [reply docs/install/install.rst]: ref: https://github.com/ROCm/rocprofiler-systems/pull/55

**Review by dgaliffiAMD — APPROVED** [review]

**Comment by peterjunpark** [comment]
@ROCm/rocm-documentation

**Review by amd-jnovotny — APPROVED** [review]
looks good.

**Pull request merged**